infra-as-code/bicep/machinelearning.bicep

```diff
@@ -159,28 +158,22 @@ resource aiHub 'Microsoft.MachineLearningServices/workspaces@2024-07-01-preview'
     friendlyName: 'Azure OpenAI Chat Hub'
     description: 'Hub to support the Microsoft Learn Azure OpenAI basic chat implementation. https://learn.microsoft.com/azure/architecture/ai-ml/architecture/basic-openai-e2e-chat'
     publicNetworkAccess: 'Enabled' // Production readiness change: The "Baseline" architecture adds ingress and egress network control over this "Basic" implementation.
-    ipAllowlist: []
-    serverlessComputeSettings: null
-    enableServiceSideCMKEncryption: false
     managedNetwork: {
-      isolationMode: 'Disabled' // Production readiness change: The "Baseline" architecture adds ingress and egress network control over this "Basic" implementation.
+      isolationMode: 'Disabled'
     }
-    allowRoleAssignmentOnRG: false // Require role assignments at the resource level.

```

Comment:
Can you let me know why this was removed.

Reply:
Same, not available anymore at 2024-10-01: https://learn.microsoft.com/en-us/azure/templates/microsoft.machinelearningservices/2024-10-01/workspaces?pivots=deployment-language-bicep

Reply:
Let's roll forward to `2025-01-01-preview` -- unless you know of a reason not to?

Reply:
This didn't get rolled forward and reverted.